### ISSUE TITLE
association_table: value for max-active-jobs rolls over or gets truncated

### ISSUE THREAD

**Issue by ryanday36** [flux-framework/flux-accounting#226]
I created a bunch of users with, for example:

```
flux account add-user --username=day36 --bank=lc --shares=1 --max-running-jobs=100 --max-active-jobs=5000
```

but the max-active-jobs apparently doesn't have enough bits to store 5000, so I end up with a 7 jobs max:

```
creation_time   mod_time        deleted         username        userid          bank            default_bank    shares          job_usage       fairshare       max_running_jobs max_active_jobs queues          
1648069813      1648070102      0               day36           36075           dev             lc              1               94203.45943999998 0.933333        100             7
```

It's not urgent, but it would probably be better to set max-active-jobs to whatever the max value is in the db if the user asks for a value greater than the max allowed.

**Comment by cmoussa1** [comment]
We talked about this a bit offline but I figured I should append what I investigated on the issue thread. I wasn't able to reproduce this issue in my Docker container and was able to successfully set a `max-active-jobs` limit of 5000 on users. I double checked the type of the column of the `association_table`:

```sql
max_active_jobs    int(11)    DEFAULT 7    NOT NULL    ON CONFLICT REPLACE DEFAULT 7,
```

The type of the column should allow integers as large as [2^63](https://stackoverflow.com/questions/4448284/sqlite3-integer-max-value), so to my knowledge, I don't think it's an overflow or truncation issue. FWIW, the default value for the limit is `7` in both the SQLite table and in the front-end Python script `flux-account.py`:

```python
subparser_add_user.add_argument(
    "--max-active-jobs",
    help="max number of both pending and running jobs",
    default=7,
    metavar="max_active_jobs",
)
```

And as you mentioned offline, you were able to successfully edit the limit to a large integer like `5000` with the `edit-user` command.  So perhaps there is an issue in the front-end command for adding the user. I'll keep investigating with the front-end tools to see if I can reproduce the issue.
